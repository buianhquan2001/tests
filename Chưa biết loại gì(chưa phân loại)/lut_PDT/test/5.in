4
10 8
   #      
   #      
   #      
   #      
 # # #    
## # #    
###### # #
##########
9 10
  ##     
  ##     
  ##     
  ## # # 
  ## # ##
 ### ####
 ### ####
 ### ####
#### ####
#########
6 8
  #   
 ###  
####  
####  
#### #
#### #
######
######
6 9
#    #
#    #
# # ##
### ##
### ##
### ##
### ##
### ##
######
