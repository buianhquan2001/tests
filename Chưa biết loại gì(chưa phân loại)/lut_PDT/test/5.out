   #      
   #      
   #      
   #      
 #-#-#    
##-#-#    
######-#-#
##########
6
  ##     
  ##     
  ##     
  ##-#-# 
  ##-#-##
 ###-####
 ###-####
 ###-####
####-####
#########
8
  #   
 ###  
####  
####  
####-#
####-#
######
######
2
#----#
#----#
#-#-##
###-##
###-##
###-##
###-##
###-##
######
15
